lib/cn/kvset.h

```diff
@@ -525,10 +525,10 @@ kc_kblock_check(struct mpool *ds, u64 kblkid, struct vb_meta *vb_meta);
  * @ds:     handle to the kvdb dataset
  * @cp:     kvs create-time params
  * @meta:   kvset_meta data
- * @khmapv: key hashmap
+ * @tree:   cn tree ptr
  */
 merr_t
-kc_kvset_check(struct mpool *ds, struct kvs_cparams *cp, struct kvset_meta *meta, u8 *khmapv);
+kc_kvset_check(struct mpool *ds, struct kvs_cparams *cp, struct kvset_meta *meta, void *tree);
```

Comment:
Any better type other than void * here?

Reply:
Aww, crap, kvck.c is polluted with knowleged of khashmap...

Reply:
Done.